server/src/main/java/org/elasticsearch/transport/RemoteClusterAware.java

```diff
@@ -48,8 +49,12 @@
     /**
      * A list of initial seed nodes to discover eligible nodes from the remote cluster
      */
-    public static final Setting.AffixSetting<List<InetSocketAddress>> REMOTE_CLUSTERS_SEEDS = Setting.affixKeySetting("search.remote.",
-        "seeds", (key) -> Setting.listSetting(key, Collections.emptyList(), RemoteClusterAware::parseSeedAddress,
+    public static final Setting.AffixSetting<List<String>> REMOTE_CLUSTERS_SEEDS = Setting.affixKeySetting("search.remote.",
```

Comment:
I am stranger here, so sorry if I am wrong, but it would be more readable:
```
public static final Setting.AffixSetting<List<String>> REMOTE_CLUSTERS_SEEDS = Setting.affixKeySetting(
    "search.remote.",
    "seeds",
    (key) -> Setting.listSetting(
        key, Collections.emptyList(),
        s -> {
            // validate seed address
            RemoteClusterAware.parseSeedAddress(s);
            return s;
        },
        Setting.Property.NodeScope, Setting.Property.Dynamic
    )
);
```

Reply:
No worries, I don't think there's much of a wrong or right here :) I tend to agree with your suggested style but if you look around the codebase you will find that my style is what's used pretty much everywhere so I went with the consistent approach here.

Reply:
Huh, I see now.
That's completely fine then.
Consistency is a very good thing, your're right.
Thanks.

Reply:
@original-brownbear I use the style suggested by @chernecov. 😇

Reply:
Hehe that's all I needed to hear :) Will reformat